.github/CODEOWNERS

```diff
@@ -89,6 +89,7 @@ exporter/tencentcloudlogserviceexporter/                 @open-telemetry/collect
 exporter/zipkinexporter/                                 @open-telemetry/collector-contrib-approvers @MovieStoreGuy @astencel-sumo @crobert-1
 
 extension/asapauthextension/                             @open-telemetry/collector-contrib-approvers @jamesmoessis @MovieStoreGuy
+extension/awsmiddleware									 @open-telemetry/collector-contrib-approvers @jefchien
```

Comment:
Is it just me or is your line not aligned with the others?

Reply:
I'm not seeing that, but it's possible.